### ISSUE TITLE
Add Option to Allow Image to be either Square or Round in Card Component

### ISSUE THREAD

**Issue by JasonFritsche** [distributeaid/distributeaid.org#601]
The header/image in the Card component is always square. Let's add the option to make it round or square.

Here is the CodeSee of files that require change: https://app.codesee.io/maps/f3d494c0-2f14-11ed-a3e9-474708ea821c

**Comment by coderbyheart** [comment]
Why is this needed?

**Comment by JasonFritsche** [comment]
@coderbyheart This issue was listed as a requirement in #547 but was not implemented in that ticket.

**Comment by coderbyheart** [comment]
Thanks for clarifying!

**Comment by JasonFritsche** [comment]
Should I expect that we will have a property coming back in the graphql query that determines if the image should be round or not?

I believe we will need to conditionally supply a class name to the imgClassName property on the GatsbyImage component itself, something like this (see imgClassName, currently no logic/nothing conditional):
```
<GatsbyImage
   key={region.name}
   image={region.map.gatsbyImageData}
   alt={`Map highlighting the ${region.name} region.`}
   className="mb-4 w-full"
   imgClassName='rounded-full'
 />
```

My initial thought was that we would wrap the `header` in the`Card` component and set the round/square styling there. However, after experimenting and reading through Gatsby's docs, I believe we will need to set the style on the `GatsbyImage` component.

**Comment by jtfairbank** [comment]
Hey @JasonFritsche!  Just getting caught up.  I believe that there will be two ways to control this behavior:

1. Property coming back from the graphql query if this component is being dynamically generated from content in the CMS.
2. Property passed in by the component's parent if the component is being statically generated in the parent.

Either way, I believe the property will be passed in by the parent and you should be able to specify how you want it structured by defining the typescript properties for this component.

Does that answer your question?

**Comment by JasonFritsche** [comment]
@jtfairbank Thanks for providing feedback. I want to clarify that we will need to set the optional round classname on either the `StaticImage` or the `GatsbyImage` and not as a prop that gets passed in to the generic `Card` component that was recently created.

**Comment by jtfairbank** [comment]
Ah I gotcha, just took a more detailed look at your implementation.  Yes, that makes sense to have it on the `StaticImage` or `GatsbyImage` that gets passed in.  So there might not be too much to do here, or it might make sense to create wrapper classes for those components that control the properties set (i.e. setup the image to be a cover for the card, or a profile-pic like circle).

Btw I believe "square" really means cover in this case, so the image extends all the way across the top of the card.  I'll ask Weston to confirm tomorrow.

**Comment by JasonFritsche** [comment]
I don't believe it's possible to[ pass in props](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#restrictions-on-using-staticimage) to the `StaticImage` component. I think we have more freedom when it comes to the `GatsbyImage` component. My inexperience with Gatsby is showing, but this leads me to think we'd have to set the cover or round styling on each of the  `StaticImage` and `GatsbyImage` instances.

**Comment by jtfairbank** [comment]
Ah yes, @JasonFritsche, you're totally right about `StaticImage`.  My guess is we'll mainly be using `GatsbyImage` and otherwise will handle creating the `StaticImage` component in the parent of the `Card` component.

**Comment by JasonFritsche** [comment]
Awesome, I'll see what I can come up with. Thanks @jtfairbank!!!

**Comment by coderbyheart** [comment]
Fixed in #644
